frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -267,14 +268,17 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
 
   const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
     return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
+      const nodeLabel = isLabelPresent(node, ocsLabel);
+      if (!nodeLabel) {
```

Comment:
Adding this condition to the 'map' callback will cause any item that doesn't meet this condition to be 'undefined' in the resulting array. So you end up with mixed 'undefined' and 'Promise' items in the array which is not desirable. I would suggest using 'reduce' instead of 'map' here.